geoapi/tests/api_tests/test_projects_service.py

```diff
@@ -46,8 +47,15 @@ def test_create_observable_project(user1,
         'watch_content': True
     }
     proj = ProjectsService.create(db_session, data, user1)
```

Comment:
Pedantic comment 8: For consistency, wouldn't it be better to use `project` instead of `proj`?

Reply:
i think either proj or project is fine. but updated to `project` in  8af0d2f6f9b6bfb4d6b3719ed5011484773c2978